implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/DynatraceMeterRegistry.java

```diff
@@ -265,8 +274,13 @@ private void postCustomMetricValues(String type, List<DynatraceTimeSeries> timeS
     }
 
     // VisibleForTesting
-    List<DynatraceBatchedPayload> createPostMessages(String type, List<DynatraceTimeSeries> timeSeries) {
-        final String header = "{\"type\":\"" + type + '\"' + ",\"series\":[";
+    List<DynatraceBatchedPayload> createPostMessages(String type, String group, List<DynatraceTimeSeries> timeSeries) {
+        final String header;
+        if (StringUtils.isNotBlank(group)) {
```

Comment:
Just a suggestion: this could be simplified with a ternary `StringUtils.isNotBlank(group) ? ... : ""` inside of the string concatenation so the other parts of the header aren't repeated. Maybe less chance of drift later.

Reply:
That's a good suggestion. Made the changes. Thanks for reviewing @shakuzen @jkschneider

Reply:
Thanks for the quick turnaround!